docs/signatures.json

```diff
@@ -1544,7 +1544,19 @@
         ]
       ],
 
-      "bugs": []
+      "bugs": [1360217]
+    },
+
+    {
+      "signatures": [
+        [
+          "getNodeText@jar:file:///C:/Program%20Files/Nightly/omni.ja!/components/TooltipTextProvider.js",
```

Comment:
One of the problems with the omni.ja stacks is that it includes a file:/// path which might not match exactly from user to user (if, for example, Firefox is not installed on a filesystem with a C:/, or within the Program Files directory).

I'll take this though. In the future, we might want to modify the add-on to do fuzzier matching for omni.ja files.